packages/workflow/src/WorkflowDataProxy.ts

```diff
@@ -699,7 +699,7 @@ export class WorkflowDataProxy {
 						' <a target="_blank" href="https://docs.n8n.io/data/data-mapping/data-item-linking/item-linking-errors/">More info</a>';
 
 					context.description += moreInfoLink;
-					context.descriptionTemplate += moreInfoLink;
+					if (context.descriptionTemplate) context.descriptionTemplate += moreInfoLink;
```

Comment:
From what I can tell testing this would require mocking nodes, connections, node types, and finding a way to check the paired error message on the last node. Since this is only unblocking an error message string, I assume all this setup for a minimal test is not worth the effort. Else let me know.

Reply:
e2e test might be okay just to have something..

Reply:
Thanks, I try to avoid adding more to e2e when the issue should be testable via unit tests like here. Ideally the paired item functionality should have a setup for this already.

Reply:
we already have good paired item e2e tests for mapping data.. but I don't think we have any tests for errors